### ISSUE TITLE
[Bug] 'module' object has no attribute 'sizeHuman'

### ISSUE THREAD

**Issue by tanatonaut** [0xInfection/TIDoS-Framework#71]
```
 [!] Type Selected : Backup Brute
 [*] Loading module...

 [*] Path to file to be used (Default: files/fuzz-db/backup_paths.lst)...
 [#] Your input (Press Enter if default) :> 
 [*] Importing wordlist...
 [-] Unknown Exception Encountered!
 [-] Exception : 'module' object has no attribute 'sizeHuman'
 [-] No backdoors were found!
 [+] Done!
```
Hello!
Can you explain root causes of this message?

**Comment by tanatonaut** [comment]
Platform Kali GNU/Linux Rolling 
Linux localhost.localdomain 4.19.0-kali4-amd64 #1 SMP Debian 4.19.28-2kali1 (2019-03-18) x86_64 GNU/Linux

python --version
Python 2.7.16rc1

**Comment by 0xInfection** [comment]
Hi this is to let you know, that this has been reproduced and a fix will be released within 10 mins. I found this issue on all other brute-force modules, so thank you so much for pulling this issue up.

**Comment by 0xInfection** [comment]
This has been addressed to and fixed in 3b2850a.

**Comment by tanatonaut** [comment]
Thanks a lot!
